flink-connectors/flink-connector-base/src/test/java/org/apache/flink/connector/base/source/reader/synchronization/FutureCompletingBlockingQueueTest.java

```diff
@@ -0,0 +1,22 @@
+package org.apache.flink.connector.base.source.reader.synchronization;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+/**
+ * The unit test for {@link FutureCompletingBlockingQueue}
+ */
+public class FutureCompletingBlockingQueueTest {
+
+	private static final Integer DEFAULT_CAPACITY = 10000;
```

Comment:
Should the unit test use a value different from the default value?

Reply:
Ok